tasks/utils/requirements.py

```diff
@@ -0,0 +1,56 @@
+# (C) Datadog, Inc. 2010-2017
```

Comment:
Should we remove these requirement specific bits since it's not packaged in the agent ?